lib/gui/models/store.js

```diff
@@ -91,7 +105,8 @@ const storeReducer = (state, action) => {
 
       const newState = state.set('availableDrives', Immutable.fromJS(action.data));
```

Comment:
These are property names declare for the default state at the top of the file. We *could* reduce the chances of typos by making constants but I wonder if its to much complexity for trying to mitigate the fact that JavaScript is not a strongly typed language. Maybe moving to TypeScript is the solution?